Makefile

```diff
@@ -28,15 +28,12 @@ clean:
 	@rm -f ${PROJECTPATH}/${CHARM_NAME}*.charm
 	@charmcraft clean
 	@echo "Remove download snap"
-	@rm -f ${PROJECTPATH}/golang-openstack-exporter*.snap
+	@rm -f ${PROJECTPATH}/charmed-openstack-exporter*.snap
```

Comment:
```suggestion
```

We should remove all references to the local snap and the `download-snap` target; these were temporary things to work with our custom built snap.

I think we also shouldn't delete a file that the user would have manually put there; it would be unexpected.

Reply:
+1, `download-snap` should be removed, and when it's removed line 30-31 should also be removed

Reply:
I forgot to remove it before. Thanks